tools/cli/commands/generate.js

```diff
@@ -236,114 +224,93 @@ export function getQuestions( type ) {
 }
 
 /**
- * Generate a package based on questions passed to it.
+ * Generate a project based on questions passed to it.
  *
  * @param {object} answers - Answers from questions.
  */
-async function generatePackage( answers = { name: 'test', description: 'n/a', buildScripts: [] } ) {
-	const project = 'packages/' + answers.name;
-	const pkgDir = path.join( __dirname, '../../..', 'projects/packages', answers.name );
+export async function generateProject(
+	answers = { name: 'test', description: 'n/a', buildScripts: [] }
+) {
+	const type = pluralize( answers.type );
+	const project = type + '/' + answers.name;
+	const projDir = path.join( __dirname, '../../..', 'projects/', type, answers.name );
 
-	createSkeleton( pluralize( answers.type ), pkgDir, answers.name );
+	createSkeleton( type, projDir, answers.name );
+
+	// Generate the composer.json file
+	if ( type !== 'github-actions' ) {
```

Comment:
Why this exclusion? We're asking questions that would change things in this file. Maybe we shouldn't be asking them, but mirror repo one still applies.